kong/plugins/jwt/handler.lua

```diff
@@ -28,6 +28,13 @@ local function retrieve_token(request, conf)
     end
   end
 
+  for _, v in ipairs(conf.cookie_names) do
+    local jwt_cookie = ngx.unescape_uri(ngx.var["cookie_" .. v])
+    if jwt_cookie and jwt_cookie ~= "" then
+      return jwt_cookie, nil
```

Comment:
No need for the second return value (`nil`) here.

Reply:
fixed.